### ISSUE TITLE
Incompatibility with ElecCore

### ISSUE THREAD

**Issue by Ligands** [Vazkii/PlayersChoice#2]
**Description**
Game crashes when loading for the first time with Player's Choice + ElecCore (required library for a few mods, such as Deep Resonance):
```
java.lang.NullPointerException
    at elec332.core.module.ModuleManager.preInitLast(ModuleManager.java:83)
    ...
```
Full crash log: https://paste.dimdev.org/fepiqevami.mccrash

I know that the stack trace is pointing to elec332's code, but technically Player's Choice could be considered the root cause as it's responsible for nuking the mods on the first-time startup; though I'm more than happy to throw the issue over the fence if this is something that should be fixed on their end! (Figured it would be valuable to bring to your attention either way.)

**Versions**
Player's Choice: 1.1-3
Forge: 14.23.4.2705

**Comment by Vazkii** [comment]
@Elec332, could you have a look at this?

**Comment by Elec332** [comment]
Yep, I think this also explains another crash report I got a few days ago.

**Comment by Vazkii** [comment]
Is this something you can fix or it is some nonsense on my end?
